business-registry-dashboard/app/pages/magicLink.vue

```diff
@@ -47,9 +47,9 @@ const createBusinessFromNR = async (business: any) => {
 onMounted(async () => {
   // Redirect unauthenticated users to login page with current URL as redirect target
   if (!isAuthenticated.value) {
-    const authWebUrl = useRuntimeConfig().public.authWebUrl
+    const registryHomeURL = useRuntimeConfig().public.registryHomeURL
     const redirectUrl = encodeURIComponent(window.location.href)
-    window.location.href = `${authWebUrl}/login?redirect=${redirectUrl}`
+    window.location.href = `${registryHomeURL}/login/?return=${redirectUrl}`
```

Comment:
Special thanks to @seeker25 for spending some time with me to figure this out.

FYI, the code I had in auth web has been reverted. We didn't need any code changes in the end.